src/OSPSuite.Core/Domain/Services/SensitivityAnalyses/SensitivityAnalysisRunResultCalculator.cs

```diff
@@ -25,28 +20,53 @@ public SensitivityAnalysisRunResultCalculator(ISensitivityAnalysisPKAnalysesTask
          _pkAnalysesTask = pkAnalysesTask;
       }
 
-      public SensitivityAnalysisRunResult CreateFor(
-         SensitivityAnalysis sensitivityAnalysis, 
-         VariationData variationData, 
-         SimulationResults simulationResults)
+      public SensitivityAnalysisRunResult CreateFor(SensitivityAnalysis sensitivityAnalysis, VariationData variationData, SimulationResults simulationResults, bool saveOutputParameterSensitivities)
       {
          var sensitivityRunResult = new SensitivityAnalysisRunResult();
 
+         addPKAnalysisSensitivities(variationData, simulationResults, sensitivityRunResult, sensitivityAnalysis);
+
+         if (saveOutputParameterSensitivities)
+            addOutputSensitivities(variationData, simulationResults, sensitivityRunResult, sensitivityAnalysis);
```

Comment:
only added if they should be return. Default for this parameter in options is false